srcpkgs/phpMyAdmin/update

```diff
@@ -0,0 +1,2 @@
+site="https://www.phpmyadmin.net/files/"
+pattern="<a href=.*>\K[\d.]+(?=</a>)"
```

Comment:
You should change this to https://www.phpmyadmin.net/home_page/version.txt
See: https://github.com/linuxserver/docker-phpmyadmin/issues/10